### PR TITLE
Build and update process refactoring

### PR DESCRIPTION
**Pull request by sergsadovyi**
- wrap strings to i18n function with proper text domain `__('string', 'alps-gutenbers-blocks')`
- add `pot` and `json` generation scripts
- refactor `latest-posts` and `accordion` code to ESNext. **TODO**: refactor other blocks

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "alps/gutenberg",
+  "type": "wordpress-plugin",
+  "version": "1.5.1",
```
Comment (designerbrent) [inline composer.json]: Will we need to update this on every new build? Can we pull this from the `kernl.version` file as part of a `npm build` script?

File: package.json [hunk 2]
```diff
@@ -1,18 +1,31 @@
 {
   "name": "alps-guten-block",
-  "version": "1.0.0",
+  "version": "1.5.1",
```
Comment (designerbrent) [inline package.json]: Same question as above update updating this version number.

File: plugin.php [hunk 3]
```diff
@@ -17,13 +17,15 @@
 	exit;
 }
 
+define('ALPS_GUTENBERG_VERSION', '1.5.1');
```
Comment (designerbrent) [inline plugin.php]: Can this version number be pulled from the `kernl.version` file as part of the `npm build` script?

**Review by designerbrent — CHANGES_REQUESTED** [review]
From my initial testing, this looks good, with the exceptions of the changes noted above.

File: src/init.php [hunk 4]
```diff
@@ -44,28 +44,26 @@ function gutenberg_blocks_block_assets() {
...
 
 	// Styles.
 	wp_enqueue_style(
-		'gutenberg-blocks-block-editor-css', // Handle.
-		plugins_url('dist/blocks.editor.build.css', dirname(__FILE__)), // Block editor CSS.
-		array('wp-edit-blocks') // Dependency to include the CSS after it.
-		// filemtime(plugin_dir_path(__DIR__) . 'dist/blocks.editor.build.css') // Version: filemtime — Gets file modification time.
+		'alps-gb-editor',
+		plugins_url('dist/blocks.editor.build.css', dirname(__FILE__)),
+		['wp-edit-blocks']
```
Comment (designerbrent) [inline src/init.php]: Make sure to include the version number for the styles.

**Comment by designerbrent** [comment]
@sergsadovyi Is this PR ready to be fully reviewed?

**Comment by sergsadovyi** [comment]
Not yet. Working on build process.

**Comment by designerbrent** [comment]
@sergsadovyi When attempting to run the code, I get the following error: 

> Fatal error: Uncaught Error: Class 'ALPS\Gutenberg\Blocks\LatestPostsBlock' not found in /Users/hardingeb/Sites/alps/alps-wordpress-3/wordpress5/wp-content/plugins/alps-gutenberg-blocks/src/init.php:47 Stack trace: #0 /Users/hardingeb/Sites/alps/alps-wordpress-3/wordpress5/wp-includes/class-wp-hook.php(287): alps_gutenberg_blocks_init('') #1 /Users/hardingeb/Sites/alps/alps-wordpress-3/wordpress5/wp-includes/class-wp-hook.php(311): WP_Hook->apply_filters(NULL, Array) #2 /Users/hardingeb/Sites/alps/alps-wordpress-3/wordpress5/wp-includes/plugin.php(478): WP_Hook->do_action(Array) #3 /Users/hardingeb/Sites/alps/alps-wordpress-3/wordpress5/wp-settings.php(540): do_action('init') #4 /Users/hardingeb/Sites/alps/alps-wordpress-3/wordpress5/wp-config.php(90): require_once('/Users/hardinge...') #5 /Users/hardingeb/Sites/alps/alps-wordpress-3/wordpress5/wp-load.php(37): require_once('/Users/hardinge...') #6 /Users/hardingeb/Sites/alps/alps-wordpress-3/wordpress5/wp-blog-header.php(13): require_once('/Users/hardinge...') #7 /Users/ in /Users/hardingeb/Sites/alps/alps-wordpress-3/wordpress5/wp-content/plugins/alps-gutenberg-blocks/src/init.php on line 47

> There has been a critical error on your website.

I tried running `npm run dev` and got the following error: 

2020-08-31 at 9 23 AM

> ./src/blocks/latest-posts/components/edit.js
> Module not found: Can't resolve '@wordpress/api-fetch' in '/Users/hardingeb/Sites/alps/alps-wordpress-3/wordpress5/wp-content/plugins/alps-gutenberg-blocks/src/blocks/latest-posts/components'

**Review by designerbrent — APPROVED** [review]
Looks good.

**Pull request merged**